clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1193,7 +1193,7 @@ void Clang::AddPreprocessingOptions(Compilation &C, const JobAction &JA,
        getToolChain().getTriple().isAMDGCN())) {
 
       // Add include/gpu-none-libc/* to our system include path. This lets us use
-      // GPU-specific system headers first. 
+      // GPU-specific system headers first.
```

Comment:
Please don't change random lines outside `clang/lib/CIR`, this might just generate unnecessary rebase noise in the future!